src/Microsoft.ApplicationInsights/Channel/IAiSerializeableTelemetry.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Microsoft.ApplicationInsights.Channel
+{
+    using System.Collections.Generic;
+
+    public interface IAiSerializeableTelemetry
```

Comment:
Spelling: Serializable does not have an 'e'
(Also change the filename)

Reply:
thanks